### ISSUE TITLE
Using keyword (set) as function name does not produce an error

### ISSUE THREAD

**Issue by rrfvtgb** [wurstscript/WurstScript#695]
**Describe the bug**
Using keyword (set) as function name does not produce an error when using it inside VSCode, or with the compiler

**Steps to reproduce the bug**
- Create a new file "testSet.wurst"
- Paste the following code
```
package testSet

function set()
    print("set()")

init
    set()
```
- (Optionnal) Run the map and enjoy the error on the blackscreen

**Comment by Frotty** [comment]
It *shouldn't* produce an error. Wurst should rename the functions automatically and never generate invalid output Jass 👍

**Comment by Cokemonkey11** [comment]
> (Optionnal) Run the map and enjoy the error on the blackscreen

> It shouldn't produce an error.

So is this a bug or not?

**Comment by peq** [comment]
The bug was that the generated `war3map.j` file contained `set` as a function name, which is not valid Jass.

In wurst code it is allowed to use `set` as an identifier, since it is not a keyword in Wurst.

**Comment by Cokemonkey11** [comment]
Sorry, I missed that there is a fix in db542b5 above. :tada:
